manifests/0000_51_olm_07_cluster_operator.yaml

```diff
@@ -10,3 +10,10 @@ status:
   versions:
   - name: operator
     version: "0.0.1-snapshot"
+  relatedObjects:
+  - group: ""
+    name: openshift-cluster-olm-operator
+    resource: namespaces
+  - group: operator.openshift.io
+    name: cluster
+    resource: OLM
```

Comment:
Same question here about `olms` vs `OLM`.

Reply:
this seems to work - though I agree, maybe the plurals would be better here. I'll update!

Thanks for calling this out!